teuthology/task/kernel.py

```diff
@@ -1111,6 +1113,11 @@ def task(ctx, config):
                     log.debug('failed to get utsrelease string using url {url}'.format(
                               url=version_url))
 
+                if not version:
+                    raise VersionNotFoundError(
+                        "{url} is invalid; contains {ver}".format(
```

Comment:
considering that version is most likely an empty string, does it make sense to print it?